.github/workflows/generate-summary.yaml

```diff
@@ -428,6 +428,19 @@ jobs:
               labels: ['staging']
             })
 
+      # Adds comment which links issue to PR
+      - name: Link Staging PR
+        uses: actions/github-script@v6
+        if: ${{ github.event == 'Staging' && github.event.pull_request }}
```

Comment:
Is there a reason you aren't using github.workflow like above?

Reply:
oops it's a typo